### PR TITLE
SVG fixes

### LINKED ISSUE #6: Exclamation icon is misspelled

### PR DESCRIPTION
**Pull request by tomgenoni**
A number of fixes
- Change color of some icons to solid black so that fill colors will work in the CSS (some were not quite #000000).
- Added `stem` to name of long right arrow.
- Changed hamburger icon to solid lines.

Fixes #6 

@danoc

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,17 @@ This file follows the format suggested by [Keep a CHANGELOG](https://github.com/
 
 ## [Unreleased][Unreleased]
 
+### Added
+
+### Changed
+- [Patch] Changed icon for hamburger to solid lines.
+
+### Fixed
+- [Patch] Fix typo for "ellipsis" and "exclamation" (#6)
```
Comment (danoc) [inline CHANGELOG.md]: This is a "Release" since people will have to update their `href`s.

Comment (danoc) [inline CHANGELOG.md]: It'd also be helpful to have the former incorrect spelling for people doing find and replace.

**Comment by danoc** [comment]
Just those two comments then LGTM.

File: CHANGELOG.md (hunk 1 above)
Comment (tomgenoni) [inline CHANGELOG.md]: Ok. FYI, the misspelling was only ever in this repo. It never made it into the app.

**Pull request merged**